lib/employees/schema.ts

```diff
@@ -36,6 +36,10 @@ export default `
     employee: Employee!
   }
 
+  input DeleteEmployeeInput {
+    id: ID!
+  }
```

Comment:
Why do we need to nest the `id` argument in an object? When deleting an employee, we would only require one parameter which is the ID, so an object shouldn't be needed